### ISSUE TITLE
Redis TTL and MaxInactiveInterval issue

### ISSUE THREAD

**Issue by closeid** [ran-jit/tomcat-cluster-redis-session-manager#9]
SessionManager.java
```java
int timeout = getContextIns().getSessionTimeout() * 60;`
int sessionTimeout = (session == null) ? 0 : session.getMaxInactiveInterval() * 60;
```
web.xml
```
<session-config>
    <session-timeout>30</session-timeout>
</session-config>
```
After session first created, redis TTL shows `108000`, why isn't `1800`?

Remove `* 60` may fix the issue.

Test env is tomcat 7 and JDK 7.

**Comment by ran-jit** [comment]
Hi @closeid,

Thank you so much for reporting this issue. I will test the same in different environments and will update you ASAP.

Thanks!

**Comment by ran-jit** [comment]
Hi @closeid  

The fix is available in 2.0.3 release. (https://github.com/ran-jit/tomcat-cluster-redis-session-manager/releases/tag/2.0.3). I request you to kindly upgrade your session manager from version x to 2.0.3.

Thanks!
